orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/helpers/CookieHelper.java

```diff
@@ -102,6 +103,20 @@ public String getClientSessionId() {
                 });
     }
 
+    public static Optional<String> parseBrowserSessionCookie(Map<String, String> headers) {
+        Optional<HttpCookie> httpCookie =
+                getHttpCookieFromRequestHeaders(headers, BROWSER_SESSION_COOKIE_NAME);
+        if (httpCookie.isEmpty()) {
+            return Optional.empty();
+        }
+
+        String[] cookieValues = httpCookie.get().getValue().split("\\.");
```

Comment:
What's this for?

Reply:
Ah yea, that's not needed. It would split "code1.code2" into ["code1", "code2"], but no need for that here.

Reply:
I don't think you've pushed this change if you've made it 🙂